### PR TITLE
Improve top navbar

### PR DESCRIPTION
**Pull request by cbrownstein-lambda**
This pull request improves the top navbar. Most significantly, this pull request:

- Changes the **Home** link in the navbar to the site's homepage.

  IMO, visitors expect "home" links to take them to the site's homepage rather than to the organization's home page. (I personally have clicked the **Home** link on several occasions expecting it to take me back to the site homepage. :smiley:)

- Changes the names of links to the names Support team uses for the pages.

- Adds external-link icons to the names of external links. (We've already had someone caught off guard when the links took them to external sites in new tabs.)

@winlambda: Please review this pull request to ensure my changes make sense from a UI perspective. Thank you.

### CODE REVIEW

**Comment by winlambda** [comment]
Changes look good and make sense to me!

**Comment by cbrownstein-lambda** [comment]
@eamonnlambda, do you mind reviewing this PR to make sure I'm implementing the changes in a sensible way? Thank you.

File: config.toml [hunk 1]
```diff
@@ -212,15 +212,17 @@ enable = false
 
 [[menu.main]]
     name = "Home"
-    weight = 1
-    url = "https://lambdalabs.com/"
```
[the remark this answers is not in the document]
Comment (cbrownstein-lambda) [reply config.toml:216]: @winlambda messaged me this about this:

> I'd be curious if people would want a way to go to our website's home in the future, but overall all the changes look good!

Are you fine with us making "link back to the Lambda homepage" a TODO?

[the remark this answers is not in the document]
Comment (eamonnlambda) [reply config.toml:216]: Yup, works for me!

**Review by eamonnlambda — APPROVED** [review]
LGTM, link back to Lambda homepage is a TODO

**Pull request merged**